src/ExtensionsSample/Samples/TwilioSamples.cs

```diff
@@ -0,0 +1,105 @@
+                To = "{CustomerPhoneNumber}",
+                From = "{StorePhoneNumber}",
+                Body = "{CustomerName}, we've received your order ({OrderId}) and have begun processing it!")]
+            out SMSMessage message)
+        {
+            // You can set additional message properties here
+            message = new SMSMessage();
+        }
+
+        /// <summary>
+        /// Demonstrates imperatively setting SMS message properties inline in the function.
+        /// </summary>
+        [Disable]
+        public static void ProcessOrder_Imperative(
+            [QueueTrigger(@"samples-orders")] Order order,
+            [TwilioSms] out SMSMessage message)
+        {
+            message = new SMSMessage()
+            {
+                From = order.StorePhoneNumber,
+                To = order.CustomerPhoneNumber,
+                Body = string.Format("{0}, we've received your order ({1}) and have begun processing it!", order.CustomerName, order.OrderId)
+            };
+        }
+
+        /// <summary>
+        /// Demonstrates the JObject binding (the JObject will be converted into an SMSMessage)
+        /// </summary>
+        [Disable]
+        public static void ProcessOrder_JObject(
+            [QueueTrigger(@"samples-orders")] Order order,
+            [TwilioSms] out JObject message)
+        {
+            message = new JObject()
+            {
+                { "From", order.StorePhoneNumber },
+                { "To", order.CustomerPhoneNumber },
+                { "Body", string.Format("{0}, we've received your order ({1}) and have begun processing it!", order.CustomerName, order.OrderId) }
+            };
+        }
+
+        /// <summary>
+        /// Demonstrates the IAsyncCollector binding. This works with JObject
+        /// or SMSMessage. Using IAsyncCollector is also a way to conditionally
+        /// send messages from a function.
+        /// </summary>
+        [Disable]
+        public static void ProcessOrder_MessageAsyncCollector(
+            [QueueTrigger(@"samples-orders")] Order order,
+            [TwilioSms] IAsyncCollector<SMSMessage> messages)
+        {
+            var message = new SMSMessage()
+            {
+                From = order.StorePhoneNumber,
+                To = order.CustomerPhoneNumber,
+                Body = string.Format("{0}, we've received your order ({1}) and have begun processing it!", order.CustomerName, order.OrderId)
+            };
+
+            messages.AddAsync(message);
+        }
+
+        /// <summary>
+        /// Demonstrates the IAsyncCollector binding. This works with JObject
+        /// or SMSMessage. Using IAsyncCollector is also a way to conditionally
+        /// send messages from a function.
+        /// </summary>
+        [Disable]
+        public static void ProcessOrder_JObjectAsyncCollector(
+            [QueueTrigger(@"samples-orders")] Order order,
+            [TwilioSms] IAsyncCollector<JObject> messages)
```

Comment:
Shouldn't SMS be capitalized in the attribute Type name? SMS is capitalized in their SDK types